### ISSUE TITLE
Improvements to HTML version of the book

### ISSUE THREAD

**Issue by ZSvedic** [ZSvedic/EBH-book#18]
Few things I noticed:

A. Footer breaks down on mobile, see the screenshot:
![Screenshot_20200123-171538](https://user-images.githubusercontent.com/43754593/73003068-47e26780-3e05-11ea-98ce-de3403a45ae1.png)
My suggestions for the footer:
- Align social icons to the left, buttons to the right.
- Add arrows (« and ») to the buttons.
- Make footer in the same style as header (blue bar).

B. Sharing of chapters doesn't work. I tried sharing the first chapter on FB, but it shared link to table of content instead (URL part after # was not shared). This is the link shared on FB:
https://svedic.org/zeljko/temp/output/ebh.html?fbclid=IwAR2AUX3R8gKcfKxx8QZEH7x8Z8zEMW4p6XVPL5_8F90Er3GlJX3R225cpkk

C. Related to B, when chapter is shared on social media, no cover image is found (see screenshot below). Check if we can add some hint to sharers to use EBH cover image as thumbnail? 
![Annotation 2020-01-23 172025](https://user-images.githubusercontent.com/43754593/73004141-15d20500-3e07-11ea-8abe-78d6d4082bae.png)

D. In header, I would move "Table of contents" button to the left (same line as text), and on the right I would align " « Previous " and " Next » " buttons from the footer.

E. We need some short introduction HTML before table of contents, that has a small thumbnail of the cover image and links to:
- EPUB download,
- Standalone PDF download,
- Amazon link for Kindle and Print editions.

**Comment by reinaldoarrosi** [comment]
I've made some improvements:

A. Even when aligning items to left/right they would still break in some resolutions. So, instead of following your suggestion, I made Prev/Next buttons break into a new line. This solves the problem for all screen resolutions.

B. When I first implemented the sharing feature links to chapters were being shared correctly, but I noticed that when sharing to Twitter it removed the # part of the URL. I've fixed it now, and it seems that it works with Facebook but not with Twitter.

C. I still haven't implemented this. I was trying to test it using Facebook debugger but I was having some trouble. I'll start working on it now.

D. This has the same problem as A: in some resolutions this layout would break (also, I think it looked a bit weird 😄). I've added Prev/Next buttons around Table of contents and made sure they break properly in small screens.

E. I've added the short intro you asked. I've copied the introduction text from Amazon, and added the download buttons. The Amazon button is a bit different since it follows Amazon's guidelines https://kdp.amazon.com/en_US/help/topic/G200652190)

I'll start working on C now. Let me know if there's anything that needs changing.
